### PR TITLE
adding initial SELinux booleans to OSPP

### PR DESCRIPTION
**Pull request by shawndwells**
Initial stab at selecting SELinux booleans for discussion in https://github.com/OpenSCAP/scap-security-guide/issues/1964

If you're trying to generate a list of rules, here is a hacked awk that worked for me:
``grep "<Rule id=\"sebool_" shared/xccdf/system/selinux.xml | awk -F"<Rule id=\"" '{ print $2 }' | awk -F"\"" '{print $1}'``

### CODE REVIEW

File: RHEL/7/input/profiles/ospp-rhel7.xml [hunk 1]
```diff
@@ -319,6 +319,67 @@ the consensus process.
...
+<select idref="sebool_cron_system_cronjob_use_shares" selected="true" />
+<select idref="sebool_cron_userdomain_transition" selected="true" />
+<select idref="sebool_daemons_dump_core" selected="true" />
+<select idref="sebool_daemons_use_tcp_wrapper" selected="true" />
+<select idref="sebool_daemons_use_tty" selected="true" />
+<select idref="sebool_deny_execmem" selected="true" />
+<select idref="sebool_deny_ptrace" selected="true" />
+<select idref="sebool_domain_fd_use" selected="true" />
+<select idref="sebool_domain_kernel_load_modules" selected="true" />
+<select idref="sebool_fips_mode" selected="true" />
```
Comment (mpreisler) [inline RHEL/7/input/profiles/ospp-rhel7.xml]: This select is already in the profile.

```
'{http://checklists.nist.gov/xccdf/1.1}select': Duplicate key-sequence ['sebool_fips_mode'] in unique identity-constraint '{http://checklists.nist.gov/xccdf/1.1}itemSelectKey'.
```

Comment (shawndwells) [inline RHEL/7/input/profiles/ospp-rhel7.xml]: Updated

Comment (shawndwells) [inline RHEL/7/input/profiles/ospp-rhel7.xml]: addressed in b645244

**Comment by redhatrises** [comment]
@shawndwells since #1971 was used, you might want to use it to help enhance this PR.

**Comment by shawndwells** [comment]
@redhatrises - Not sure that's needed, since the default XCCDF values reflect default system values, which is what we want.

**Comment by mpreisler** [comment]
ACK

**Pull request merged**